manim/scene/scene_file_writer.py

```diff
@@ -77,8 +78,10 @@ def init_output_directories(self, scene_name):
                 image_dir = guarantee_existence(
                     config.get_dir("images_dir", module_name=module_name)
                 )
-            self.image_file_path = os.path.join(
-                image_dir, add_extension_if_not_present(default_name, ".png")
+            self.image_file_path = Path(
+                os.path.join(
+                    image_dir, add_extension_if_not_present(default_name, ".png")
+                )
             )
```

Comment:
To remove unnecessary `os.path` usage:
```suggestion
            self.image_file_path = Path(image_dir, add_extension_if_not_present(default_name, ".png"))
```

Reply:
(probably not black-conforming)

Reply:
I'd tested this before, and it wouldn't work without it. But it does now with some other changes.